packaging/google-osconfig-agent.conf

```diff
@@ -0,0 +1,5 @@
+description "Google OSConfig Agent"
```

Comment:
you were going to remove these files from the PR, right?

Reply:
removed it